packages/twenty-front/src/modules/activities/calendar/components/CalendarEventRow.tsx

```diff
@@ -48,14 +48,15 @@ const StyledAttendanceIndicator = styled.div<{ active?: boolean }>`
 const StyledLabels = styled.div`
   align-items: center;
   display: flex;
-  color: ${({ theme }) => theme.font.color.tertiary};
+  color: ${({ theme }) => theme.font.color.primary};
```

Comment:
based on the issue, it seems that we want secondary and not primary?

Reply:
I had a back and forth conversation with another member of Twenty in the comments of the original issue who said that primary looked better from the screenshots I posted.

Reply:
Ok, I missed it, LGTM!